Dockerfile.fedora

```diff
@@ -2,22 +2,19 @@ FROM fedora
 MAINTAINER langdon <langdon@fedoraproject.org>
 #Derived from Vaclav Pavlin <vpavlin@redhat.com>; https://github.com/vpavlin/atomicapp-run
 
-RUN yum -y update && yum clean all
-RUN yum -y install python python-pip docker && yum clean all
-
-ADD requirements.txt /opt/atomicapp/
-ADD setup.py /opt/atomicapp/
-ADD atomicapp/ /opt/atomicapp/atomicapp/
-
-WORKDIR /opt/atomicapp
-RUN python setup.py install
+RUN yum -y update
+RUN yum -y install python python-pip docker-io kubernetes && yum clean all
 
+COPY . /opt/atomicapp/
+WORKDIR /opt/atomicapp/
+RUN chmod u+x /opt/atomicapp/setup.py
```

Comment:
This is useless layer - I don't think it makes any difference to be able to call ./setup.py instead of python setup.py